07_memory/user_mem/logs.rst

```diff
@@ -0,0 +1,237 @@
+        24      0               40              56              |       0               0               0               16777216
+        25      0               40              56              |       0               0               0               33554432
+        26      0               40              56              |       0               0               0               67108864
+        27      0               40              56              |       0               0               0               134217728
+        28      0               41              56              |       0               0               0               268435456
+        29      0               40              56              |       0               0               0               536870912
+        30      0               40              56              |       0               0               0               1073741824
+        31      0               40              56              |       0               0               0               2147483648
+        32      0               41              56              |       0               0               0               4294967296
+        33      0               41              56              |       0               0               0               8589934592
+        34      0               40              56              |       0               0               0               17179869184
+        35      0               40              56              |       0               0               0               34359738368
+        36      0               41              56              |       0               0               0               68719476736
+        37      0               40              56              |       0               0               0               137438953472
+        38      0               41              56              |       0               0               0               274877906944
+        39      0               40              56              |       0               0               0               549755813888
+        40      0               40              56              |       0               0               0               1099511627776
+        41      0               40              56              |       0               0               0               2199023255552
+        42      0               41              56              |       0               0               0               4398046511104
+        43      0               40              56              |       0               0               0               8796093022208
+        44      0               40              56              |       0               0               0               17592186044416
+        45      0               40              56              |       0               0               0               35184372088832
```

Comment:
I suppose it is impossible to allocate so huge buffer on your computer. Please change your code.
We alternative with segmentation fault is better than completely wrong results.